src/add_project_handler.py

```diff
@@ -167,6 +169,25 @@ async def add_project(update: Update, context: ContextTypes.DEFAULT_TYPE):
         parse_mode=ParseMode.MARKDOWN_V2,
     )
 
+    message = student_message.text
+    check_add_data = google_sheets_package.connect_modules_to_add_data_to_gsheets(
+        message=message,
+        lang_project=language,
+        type_project=project_name
+    )
+
+    if check_add_data.boolean_val is False:
+        error_message = await context.bot.send_message(
+            chat_id=chat_id,
+            text=check_add_data.error_message,
+            reply_to_message_id=student_message.id,
+        )
+        await asyncio.sleep(15)
+        await context.bot.delete_messages(
+            chat_id=chat_id,
+            message_ids=[error_message.id]
+        )
```

Comment:
- Сообщение с ошибкой пойдет юзеру, а должно идти тому кто вызвал команду
- sleep нужен на 10 секунд, как в остальных случаях
- Удалить нужно 2 сообщения - об ошибке + вызов команды

```suggestion
    if check_add_data.boolean_val is False:
        error_message = await context.bot.send_message(
            chat_id=chat_id,
            text=check_add_data.error_message,
            reply_to_message_id=update.effective_message.id,
        )
        await asyncio.sleep(10)
        await context.bot.delete_messages(
            chat_id=chat_id,
            message_ids=[error_message.id, update.effective_message.id]
        )
```